src/main/java/org/codehaus/plexus/archiver/AbstractUnArchiver.java

```diff
@@ -278,8 +280,9 @@ protected void extractFile(
             String entryName,
             final Date entryDate,
             final boolean isDirectory,
+            final boolean isSymlink,
```

Comment:
This is breaking change. I wonder if we can avoid it.

Reply:
I don't see how.